.github/workflows/test.yml

```diff
@@ -407,11 +407,12 @@ jobs:
           ref: ${{ github.ref }}
           fetch-depth: ${{ steps.fetch-depth.outputs.depth  }}
 
-      - name: Build turborepo CLI from source
-        uses: ./.github/actions/setup-turborepo-environment
+      - uses: ./.github/actions/setup-turborepo-environment
```

Comment:
all other uses of this shared action are nameless. also this name is incorrect because we aren't actually building from source anymore in there